Learning.md

```diff
@@ -1,8 +1,34 @@
- What I've learnt so far
+# What I've learnt so far
 
-* Commandline
-* Intro to Git & Github
-* Intro to software development
+- [What I've learnt so far](#what-ive-learnt-so-far)
+  - [What I like so far about the course](#what-i-like-so-far-about-the-course)
+  - [Some of the commandline command that has been covered include:](#some-of-the-commandline-command-that-has-been-covered-include)
+  - [Some of the Git command that has been covered include:](#some-of-the-git-command-that-has-been-covered-include)
+  - [Intro to software development](#intro-to-software-development)
+  - [What I found challenging](#what-i-found-challenging)
 
 ## What I like so far about the course
 I like the method of teaching and how helpful the instructors are.
+
+## Some of the commandline command that has been covered include:
+*  **Cd**
+* **mkdir** 
+* **mv** 
+* **ls** 
+
+## Some of the Git command that has been covered include:
+* **git clone**
+* **git push** 
+* **git add [code to add]** 
+* **git checkout [branch to checkout]** 
+* **git pull** 
+* **git commit -m "[commit message]"** 
+* **git branch [branch to create]** 
```

Comment:
Commands are typically rendered as inline code in Markdown, you could update this formatting to follow convention?

Reply:
great, thank you!